### DISCUSSION TITLE
Custom domain with godaddy

### DISCUSSION THREAD

**Discussion by Darshancsagar** [vercel/community#3912] [Help] [answered]
### Summary

I have custom domain linked to my next.js app but my custom domain not working even after setting name servers in godaddy.com.
while opening my site in custom domain it shows like "ERR_CONNECTION_CLOSED".

But when i use VPN i can open my site.

Please tell me solution

### Steps to Reproduce

1.Create a next.js project.
2.Add custom domain from godaddy.com
3.lunch your site for custom domain.

**Comment by Okpedan** [comment]
Am experiencing same thing here

**Comment by amyegan** [comment, marked as answer]
Please go through these troubleshooting steps to verify that your domain is configured correctly: https://vercel.com/docs/projects/domains/troubleshooting 🙏

**Reply by Darshancsagar** [reply]
Site is working but its not opening when i connect my device to a particular wifi.
It will work for other wifi networks.
